### DISCUSSION TITLE
Welcome to HybPiper Discussions!

### DISCUSSION THREAD

**Discussion by mossmatters** [mossmatters/HybPiper#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Eremus007** [comment]
Hey, my name is Jeremy, and I am currently looking to wrap Hybpiper in a tool for GalaxyProject to make Hybpiper accessible to people without knowledge of a commandline interface. 
https://github.com/naturalis/galaxy-pipeline-hybseq/tree/development/tools/hybpiper

My question is, is there a way to pass an argument to Hybpiper of the directory in which they want Hybpiper's output files to go?

If there is not, as a suggestion, I would love to see something like a -o/--hybpiper_output flag/subparser where users can specify this and redirect the output.

I have learned that, at the moment, Hybpiper creates its output directories in the same location as the read files, but this complicates matters for me in Galaxy, as Galaxy does not like things being created in its specialized folders.

Would love to hear about your thoughts on this.

**Reply by Eremus007** [reply]
Once I get things working, will of course link back to this original repo with proper credits ^^

**Reply by mossmatters** [reply]
Jeremy - happy to hear HybPiper is heading for GalaxyProject! We're happy to consider quality of life changes like you suggest if it is going to make the implementation easier. Can you please re-post your `-o` request over at the issues page? https://github.com/mossmatters/HybPiper/issues That way all the devs and users can see it in one place. Thanks!
